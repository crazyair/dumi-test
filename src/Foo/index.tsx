import { Button } from 'antd';
import React from 'react';

const Foo = () => {
  return <Button>xxx</Button>;
};

export default Foo;
