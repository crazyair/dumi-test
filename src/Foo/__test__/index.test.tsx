import { render } from '@testing-library/react';
import React from 'react';

import Foo from '..';

describe('test props', () => {
  it('render', () => {
    const { asFragment } = render(<Foo title="bbb" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
