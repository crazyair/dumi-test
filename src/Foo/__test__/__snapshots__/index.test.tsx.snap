// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test props render 1`] = `
<DocumentFragment>
  <button
    class="ant-btn ant-btn-default"
    type="button"
  >
    <span>
      xxx
    </span>
  </button>
</DocumentFragment>
`;
